src/Header/Header.js

```diff
@@ -3,8 +3,35 @@ import "./Header.css";
 
 class Header extends React.Component {
   render() {
-    return <div>I am a top bar. please fix me</div>;
+    return (<div className="head">
+      <div className="greeting-row">
+        <span className="greeting">Welcome Username!</span>
+      </div>
+      <div className="button-row">
+        <div className="buttons">
+          <button
+            className="share"
+            onClick={() => alert('share link copied')}>
+            <span className="text1">
+              Share
+         </span>
+          </button>
+          <button
+            className="AR"
+            onClick={() => alert('more features to come')}>
+            <span className="text2">
+              AR
+         </span>
+          </button>
+        </div>
+        <button
```

Comment:
I'm not actually sure if the profile is meant to be clickable.
It looks like the designers may want it to eventually be a picture, so it might be worth it to swap this `button` element for a `div` or `img` tag (which simplifies the CSS a bit, since you wouldn't have to override `button`'s default styles).